src-plugins/mscAlgorithmPaint/mscAlgorithmPaintToolBox.cpp

```diff
@@ -578,25 +571,24 @@ void AlgorithmPaintToolBox::activateMagicWand()
 
 void AlgorithmPaintToolBox::behaveWithBodyVisibility()
```

Comment:
Indeed, it's called when you switch from a toolbox to another. It allows to hide/disable buttons on specific toolboxes if you need, as in algorithmPaint or polygonROI (if you remember, before this PR, we had a specific function in `medSegmentationSelectorToolBox` which was a small hack but not very clean).